### PR TITLE
Fairly minimal .vsconfig

### PR DESCRIPTION
**Pull request by robertpi**
Confused about which C++ to check?

Don't be any more!

Fixes #

Changes proposed in this pull request:

@DataDog/apm-dotnet

### CODE REVIEW

**Review by andrewlock — APPROVED** [review]
Excellent! How did you generate this btw? I assume VS prompts you to install missing components when you open the sln if it finds this file?

**Comment by robertpi** [comment]
> How did you generate this btw? 

I exported my settings, which you can also do via "Visual Studio Installer", although the functionality is quite well hidden. (That's why it's "fairly minimal", they may be some stuff that isn't need). 

I took my settings from [dotnet/runtime sometime agi](https://github.com/dotnet/runtime/blob/main/.vsconfig), it looks like they're pretty similar.

> I assume VS prompts you to install missing components when you open the sln if it finds this file?

Sadly not, but not a lot we can do about that.

**Comment by andrewlock** [comment]
## Benchmarks Report :snail:

Benchmarks for #2401 compared to [master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3): 
* 1 benchmarks are **faster**, with geometric mean 1.196
* All benchmarks have **the same** allocations


The following thresholds were used for comparing the benchmark speeds:
* Mann–Whitney U test with statistical test for significance of **5%**
* Only results indicating a difference greater than **10%** and **0.3 ns** are considered.

Allocation changes below **0.5%** are ignored.


### Benchmark details


<details><summary><strong>Benchmarks.Trace.AgentWriterBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`WriteAndFlushEnrichedTraces`|net472|837μs|3.49μs|13.5μs|0| 0|0|3.16&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|649μs|3.1μs|12μs|0| 0|0|2.57&nbsp;KB|
|#2401|`WriteAndFlushEnrichedTraces`|net472|837μs|3.21μs|12μs|0| 0|0|3.16&nbsp;KB|
|#2401|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|645μs|1.99μs|7.71μs|0| 0|0|2.57&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AspNetCoreBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`SendRequest`|netcoreapp3.1|291μs|799ns|2.99μs|0.149| 0|0|19.67&nbsp;KB|
|#2401|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|#2401|`SendRequest`|netcoreapp3.1|292μs|1.34μs|6.45μs|0.142| 0|0|19.67&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.DbCommandBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`ExecuteNonQuery`|net472|1.57μs|6.93ns|25.9ns|0.0926| 0.000772|0|586&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`ExecuteNonQuery`|netcoreapp3.1|1.45μs|5.84ns|22.6ns|0.00842| 0|0|624&nbsp;B|
|#2401|`ExecuteNonQuery`|net472|1.63μs|8.7ns|47.7ns|0.093| 0.000788|0|586&nbsp;B|
|#2401|`ExecuteNonQuery`|netcoreapp3.1|1.44μs|8ns|48.7ns|0.00848| 0|0|624&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.ElasticsearchBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`CallElasticsearch`|net472|2.28μs|10.3ns|39.9ns|0.124| 0|0|794&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`CallElasticsearch`|netcoreapp3.1|1.68μs|9.43ns|61.1ns|0.0106| 0|0|784&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`CallElasticsearchAsync`|net472|2.52μs|14.1ns|89.1ns|0.145| 0|0|931&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`CallElasticsearchAsync`|netcoreapp3.1|1.84μs|5.08ns|18.3ns|0.012| 0|0|904&nbsp;B|
|#2401|`CallElasticsearch`|net472|2.35μs|11.6ns|50.7ns|0.124| 0|0|794&nbsp;B|
|#2401|`CallElasticsearch`|netcoreapp3.1|1.59μs|7.81ns|34.9ns|0.011| 0|0|784&nbsp;B|
|#2401|`CallElasticsearchAsync`|net472|2.42μs|3.91ns|14.1ns|0.146| 0|0|931&nbsp;B|
|#2401|`CallElasticsearchAsync`|netcoreapp3.1|1.77μs|8.42ns|34.7ns|0.0123| 0|0|904&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.GraphQLBenchmark</strong> - Faster :tada: Same allocations :heavy_check_mark:



#### Faster :tada: in #2401

|  Benchmark         | base/diff | Base Median (ns) | Diff Median (ns) | Modality   |
|:----------|-----------:|-----------:|--------:|--------:|
| Benchmarks.Trace.GraphQLBenchmark.ExecuteAsync&#8209;net472 | 1.196 | 3,192.14 | 2,668.34 | 

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`ExecuteAsync`|net472|3.21μs|16ns|67.9ns|0.165| 0|0|1.05&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`ExecuteAsync`|netcoreapp3.1|1.91μs|8.77ns|34ns|0.0135| 0|0|1.02&nbsp;KB|
|#2401|`ExecuteAsync`|net472|2.68μs|10.5ns|37.7ns|0.165| 0|0|1.05&nbsp;KB|
|#2401|`ExecuteAsync`|netcoreapp3.1|1.89μs|6.1ns|23.6ns|0.0138| 0|0|1.02&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.HttpClientBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`SendAsync`|net472|6.13μs|28ns|108ns|0.352| 0|0|2.26&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`SendAsync`|netcoreapp3.1|4.35μs|11.9ns|45.9ns|0.0303| 0|0|2.16&nbsp;KB|
|#2401|`SendAsync`|net472|6.2μs|31.9ns|153ns|0.353| 0|0|2.26&nbsp;KB|
|#2401|`SendAsync`|netcoreapp3.1|4.26μs|15.7ns|56.6ns|0.0301| 0|0|2.16&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.ILoggerBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`EnrichedLog`|net472|3.26μs|16.1ns|70.4ns|0.225| 0|0|1.44&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`EnrichedLog`|netcoreapp3.1|3.08μs|9.67ns|36.2ns|0.0214| 0|0|1.52&nbsp;KB|
|#2401|`EnrichedLog`|net472|3.18μs|6.4ns|22.2ns|0.226| 0|0|1.44&nbsp;KB|
|#2401|`EnrichedLog`|netcoreapp3.1|3.06μs|8.98ns|32.4ns|0.0214| 0|0|1.52&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.Log4netBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`EnrichedLog`|net472|287μs|1.2μs|4.34μs|0.429| 0.143|0|4.31&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`EnrichedLog`|netcoreapp3.1|236μs|584ns|2.11μs|0| 0|0|4.19&nbsp;KB|
|#2401|`EnrichedLog`|net472|286μs|1.3μs|5.04μs|0.438| 0.146|0|4.31&nbsp;KB|
|#2401|`EnrichedLog`|netcoreapp3.1|235μs|1.24μs|6.46μs|0| 0|0|4.19&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.NLogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`EnrichedLog`|net472|7.38μs|25.5ns|98.8ns|0.504| 0|0|3.21&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`EnrichedLog`|netcoreapp3.1|6.31μs|34.8ns|220ns|0.0491| 0|0|3.58&nbsp;KB|
|#2401|`EnrichedLog`|net472|7.46μs|38.9ns|213ns|0.502| 0|0|3.21&nbsp;KB|
|#2401|`EnrichedLog`|netcoreapp3.1|6.17μs|29.1ns|120ns|0.0491| 0|0|3.58&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.RedisBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`SendReceive`|net472|2.11μs|11.2ns|54.8ns|0.157| 0|0|1&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`SendReceive`|netcoreapp3.1|1.97μs|5.61ns|21ns|0.0137| 0|0|1&nbsp;KB|
|#2401|`SendReceive`|net472|2.27μs|6.29ns|24.4ns|0.157| 0|0|1&nbsp;KB|
|#2401|`SendReceive`|netcoreapp3.1|1.92μs|6ns|23.3ns|0.0143| 0|0|1&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SerilogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`EnrichedLog`|net472|5.85μs|27.5ns|106ns|0.288| 0|0|1.85&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`EnrichedLog`|netcoreapp3.1|5.05μs|19ns|71.1ns|0.0199| 0|0|1.47&nbsp;KB|
|#2401|`EnrichedLog`|net472|5.9μs|28.4ns|139ns|0.289| 0|0|1.85&nbsp;KB|
|#2401|`EnrichedLog`|netcoreapp3.1|5.21μs|21.7ns|83.9ns|0.0202| 0|0|1.47&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SpanBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`StartFinishSpan`|net472|905ns|3.37ns|13.1ns|0.0703| 0|0|449&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`StartFinishSpan`|netcoreapp3.1|873ns|3.7ns|13.3ns|0.00643| 0|0|448&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`StartFinishScope`|net472|1.08μs|4.51ns|17.5ns|0.083| 0|0|530&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3)|`StartFinishScope`|netcoreapp3.1|1.01μs|4.74ns|18.9ns|0.00803| 0|0|568&nbsp;B|
|#2401|`StartFinishSpan`|net472|903ns|3.48ns|13ns|0.0706| 0|0|449&nbsp;B|
|#2401|`StartFinishSpan`|netcoreapp3.1|864ns|4.03ns|15.6ns|0.00603| 0|0|448&nbsp;B|
|#2401|`StartFinishScope`|net472|1.05μs|5.09ns|20.3ns|0.0832| 0|0|530&nbsp;B|
|#2401|`StartFinishScope`|netcoreapp3.1|1.02μs|4.79ns|18.6ns|0.00786| 0|0|568&nbsp;B|
</details>

**Comment by andrewlock** [comment]
## Code Coverage Report :bar_chart:

:heavy_check_mark: Merging #2401 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3) will **not change** line coverage
:heavy_check_mark: Merging #2401 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3) will **not change** branch coverage
:heavy_check_mark: Merging #2401 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3) will **not change** complexity


|           | [master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3) | #2401       | Change   | 
|:----------|:-----------:|:-----------:|:--------:|
| Lines     | `10432` / `14873` | `10392` / `14873` |          |
| Lines %   | `70%`      | `70%`      |  `0%` :heavy_check_mark:  |
| Branches  | `5341` / `7780` | `5320` / `7780` |          |
| Branches %| `69%`      | `68%`      |  `0%` :heavy_check_mark:  |
| Complexity|   `8927`      | `8927`        |  `0`  :heavy_check_mark:    |

View the full report for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/$dd-trace-dotnet/_build/results?buildId=89225&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/89225/artifacts?artifactName=Code%20Coverage%20Report_89225&api-version=6.0&%24format=zip)  
* [HTML report for this PR #2401](https://dev.azure.com/datadoghq/$dd-trace-dotnet/_build/results?buildId=89230&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/89230/artifacts?artifactName=Code%20Coverage%20Report_89230&api-version=6.0&%24format=zip)

### Datadog.Trace Breakdown :heavy_check_mark:

|        | [master](https://github.com/DataDog/dd-trace-dotnet/tree/908b4230e6969c39fba7e4b0229744228c2d1cf3) | #2401       | Change   | 
|:-------|:-----------:|:-----------:|:--------:|
| Lines %| `70%`      | `70%`       |  `0%` :heavy_check_mark:  |
| Branches %| `69%`      | `68%`       |  `0%` :heavy_check_mark:  |
| Complexity| `8927`      | `8927`       |  `0` :heavy_check_mark:  |

The following classes have significant coverage changes.

| File    | Line coverage change | Branch coverage change | Complexity change |
|:--------|:--------------------:|:----------:|:--------:|
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Couchbase.IIOServiceExecuteIntegrationBis](https://github.com/DataDog/dd-trace-dotnet/tree/3edec7320cb1e3a2dc718725f89bd1dc0a5791e7/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Couchbase/IIOServiceExecuteIntegrationBis.cs) | `-100%` :no_entry: | `0%` :heavy_check_mark:   | `0` :heavy_check_mark: |
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2.TestMethodRunnerExecuteIntegration](https://github.com/DataDog/dd-trace-dotnet/tree/3edec7320cb1e3a2dc718725f89bd1dc0a5791e7/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestMethodRunnerExecuteIntegration.cs) | `-33%` :no_entry: | `-19%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.NUnit.NUnitCompositeWorkItemSkipChildrenIntegration](https://github.com/DataDog/dd-trace-dotnet/tree/3edec7320cb1e3a2dc718725f89bd1dc0a5791e7/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/NUnitCompositeWorkItemSkipChildrenIntegration.cs) | `-8%` :no_entry: | `-8%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Ci.Agent.CIAgentWriter](https://github.com/DataDog/dd-trace-dotnet/tree/3edec7320cb1e3a2dc718725f89bd1dc0a5791e7/tracer/src/Datadog.Trace/Ci/Agent/CIAgentWriter.cs) | `-5%` :no_entry: | `-21%` :no_entry:   | `0` :heavy_check_mark: |


View the full reports for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/$dd-trace-dotnet/_build/results?buildId=89225&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/89225/artifacts?artifactName=Code%20Coverage%20Report_89225&api-version=6.0&%24format=zip)  
* [HTML report for this PR #2401](https://dev.azure.com/datadoghq/$dd-trace-dotnet/_build/results?buildId=89230&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/89230/artifacts?artifactName=Code%20Coverage%20Report_89230&api-version=6.0&%24format=zip)

**Comment by robertpi** [comment]
> 
> > I assume VS prompts you to install missing components when you open the sln if it finds this file?
> 
> Sadly not, but not a lot we can do about that.

@anna-git informs that VS does prompt you, so all good!

**Comment by anna-git** [comment]
> > How did you generate this btw?
> 
> I exported my settings, which you can also do via "Visual Studio Installer", although the functionality is quite well hidden. (That's why it's "fairly minimal", they may be some stuff that isn't need).
> 
> I took my settings from [dotnet/runtime](https://github.com/dotnet/runtime/blob/main/.vsconfig), it looks like they're pretty similar.
> 
> > I assume VS prompts you to install missing components when you open the sln if it finds this file?
> 
> Sadly not, but not a lot we can do about that.

It actually prompted me and helped build the tracer on my new machine so thanks :)

**Review by anna-git — APPROVED** [review]

File: .vsconfig [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    "Microsoft.VisualStudio.Component.VC.Tools.ARM",
+    "Microsoft.Net.ComponentGroup.TargetingPacks.Common",
+    "Microsoft.Net.Component.4.6.TargetingPack",
+    "Microsoft.VisualStudio.Component.Git",
+    "Microsoft.VisualStudio.Component.LinqToSql",
+    "Microsoft.NetCore.Component.Runtime.3.1",
+    "Microsoft.NetCore.Component.Runtime.5.0",
+    "Microsoft.Net.Component.4.6.1.TargetingPack",
+    "Microsoft.Net.Component.4.6.2.SDK",
+    "Microsoft.Net.Component.4.7.SDK"
```
Comment (lucaspimentel) [inline .vsconfig:84]: For the .NET Framework SDKs, we only need `Microsoft.Net.Component.4.8.SDK` as long as we have the other targeting packs (4.6.1, etc).

File: .vsconfig [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
+    "Microsoft.VisualStudio.Component.Windows10SDK.19041",
+    "Microsoft.VisualStudio.Component.ManagedDesktop.Prerequisites",
+    "Microsoft.ComponentGroup.Blend",
+    "Microsoft.VisualStudio.Component.FSharp.Desktop",
+    "Microsoft.VisualStudio.ComponentGroup.MSIX.Packaging",
+    "Microsoft.VisualStudio.Workload.ManagedDesktop",
+    "Microsoft.VisualStudio.Component.VC.Redist.14.Latest",
+    "Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core",
+    "Microsoft.VisualStudio.Component.VC.Tools.ARM64",
```
Comment (lucaspimentel) [inline .vsconfig:71]: We don't support ARM or ARM64 on Windows.

**Pull request merged**